### ISSUE TITLE
Add a screenshot

### ISSUE THREAD

**Issue by 0x48piraj** [fossasia/susi_desktop#95]
**Actual Behaviour**
Repo lacks a screenshot.

**Expected Behaviour**
It should have one!

##### Please check the applicable fields -

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this). or @niteshkumarniranjan :heart: 
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.
